runtime/src/test/scala/com/soundcloud/twinagle/ServerSpec.scala

```diff
@@ -13,13 +13,13 @@ import scala.collection.mutable.ListBuffer
 
 class ServerSpec extends Specification with Mockito {
   trait Context extends Scope {
-    val rpc = mock[TestMessage => Future[TestMessage]]
-    val protoService = ProtoService(
+    val rpc: TestMessage => Future[TestMessage] = mock[TestMessage => Future[TestMessage]]
```

Comment:
just wondering (and ditto for the other specs): do we need to duplicate the types or would type inference make something like this work?
```suggestion
    val rpc: TestMessage => Future[TestMessage] = mock
```

alternatively, could we drop the type annotations if we restrict the visibility? Something like
```suggestion
    protected val rpc  = mock[TestMessage => Future[TestMessage]]
```